src/components/utils/YodaMeta.js

```diff
@@ -0,0 +1,38 @@
+class YodaMeta {
```

Comment:
Broad-naming classes like this is always an invitation for ugly code in the future. Because the class (or the file itself) is so generic (and it actually covers both LOGIN_TYPE vars, URL vars, and error codes), pretty soon people will start dumping all kinds of constants in this file.

I suggest breaking this up into 3 classes (1 for LOGIN, 1 for URLs, 1 for error codes), and naming them appropriately (LoginMeta.js, UrlMeta, ErrorMeta). 

In fact, creating a directory like "util" is problematic (because everything will go into that directory), but that's another story. :-)